blendergltf.py

```diff
@@ -946,29 +946,51 @@ def png_pack(png_tag, data):
         png_pack(b'IDAT', zlib.compress(raw_data, 9)),
         png_pack(b'IEND', b'')])
 
-    return 'data:image/png;base64,' + base64.b64encode(png_bytes).decode()
+    if bytes:
+        return png_bytes
+    else:
+        return 'data:image/png;base64,' + base64.b64encode(png_bytes).decode()
 
 
 def export_images(settings, images):
     def check_image(image):
         errors = []
         if image.size[0] == 0:
-            errors.add('x dimension is 0')
+            errors.append('x dimension is 0')
         if image.size[1] == 0:
-            errors.add('y dimension is 0')
+            errors.append('y dimension is 0')
+        if image.type != 'IMAGE':
+            errors.append('not an image')
 
-        if errors:
+        if len(errors) > 0:
```

Comment:
Any particular reason to change this to a length check? I would prefer to just stick with "if errors" since I think this is my Pythonic, but I can be convinced otherwise.